MapboxNavigation/NavigationViewController.swift

```diff
@@ -296,25 +296,36 @@ public class NavigationViewController: UIViewController {
             styleManager.automaticallyAdjustsStyleForTimeOfDay = automaticallyAdjustsStyleForTimeOfDay
         }
     }
-    
+
     var mapViewController: RouteMapViewController?
-    
+
     /**
      A Boolean value that determines whether the map annotates the locations at which instructions are spoken for debugging purposes.
      */
     @objc public var annotatesSpokenInstructions = false
-    
+
     /**
      A Boolean value that indicates whether the dark style should apply when a route controller enters a tunnel.
      */
     @objc public var usesNightStyleInsideTunnels: Bool = false
     
+    /**
+     If true, the maximum speed limit for the segment of road the user is current traveling on will be shown.
+     */
+    @objc public var showMaximumSpeedLimitSign = false {
```

Comment:
Instead of this extra option on NavigationViewController, show the speed limit sign if the route options specifies the `maximumSpeedLimit` attribution option. Don’t add this attribute option automatically; instead, require the developer to add it to NavigationRouteOptions explicitly.

Reply:
I think we have a good pattern right now of not making developers mutate `NavigationRouteOptions`. I see your logic, but I'm also thinking of time when we use max speeds in other parts of sdk.

Reply:
So until then, developers who leave this setting `false` will nonetheless request speed limit attributes from the server?